pkg/schedule/filter/comparer.go

```diff
@@ -40,6 +41,25 @@ func RegionScoreComparer(conf config.SharedConfigProvider) StoreComparer {
 	}
 }
 
+// LeaderScoreComparer creates a StoreComparer to sort store by leader
+// score.
+func LeaderScoreComparer(conf config.SharedConfigProvider) StoreComparer {
+	leaderSchedulePolicy := conf.GetLeaderSchedulePolicy()
+	return func(a, b *core.StoreInfo) int {
+		// TODO: we should use the real time delta data to calculate the score.
+		sa := a.LeaderScore(leaderSchedulePolicy, 0)
```

Comment:
The leader score is not very accurate, so it leads to the leader count of the lowest score goes up too much. How about considering the the running operators influence？

Reply:
done